### PR TITLE
MODFISTO-505. Prepare for Ramsons release

### PR DESCRIPTION
**Pull request by SerhiiNosko**
## Purpose
https://folio-org.atlassian.net/browse/MODFISTO-505
Prepare for Ramsons release

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,25 @@
...
+* [MODFISTO-487](https://folio-org.atlassian.net/browse/MODFISTO-458) Prevent sql injection for budget_encumbrance_rollover script
+* [MODFISTO-483](https://folio-org.atlassian.net/browse/MODFISTO-483) Add credited field support to Rollover functionality
+* [MODFISTO-466](https://folio-org.atlassian.net/browse/MODFISTO-466) New transactionType indices
+* [MODFISTO-460](https://folio-org.atlassian.net/browse/MODFISTO-460) Remove the old transaction API (storage)
+* [MODFISTO-371](https://folio-org.atlassian.net/browse/MODFISTO-371) Improved cashBalance, available and unavailable calculations
+
+### Bug fixes
+* [MODFISTO-488](https://folio-org.atlassian.net/browse/MODFISTO-488) Fixed encumbrance changes after cancelling a payment or credit
+* [MODFISTO-484](https://folio-org.atlassian.net/browse/MODFISTO-484) Fixed overEncumbrance calculation
+* [MODFISTO-483](https://folio-org.atlassian.net/browse/MODFISTO-480) RMB bug workaround to delete tables
```
Comment (azizbekxm) [inline NEWS.md:18]: * [MODFISTO-480] RMB bug workaround to delete tables

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,25 @@
-## 8.7.0 - Unreleased
```
Comment (azizbekxm) [inline NEWS.md:1]: Also would be good to include next unreleased version

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,25 @@
...
+## 8.7.0 - Released (Ramsons R2 2024)
+The primary focus of this release was to separate Credited from Expended and improve different calculations
+[Full Changelog](https://github.com/folio-org/mod-finance-storage/compare/v8.6.0...v8.7.0)
+
+### Stories
+* [MODFISTO-499](https://folio-org.atlassian.net/browse/MODFISTO-499) Update libraries of dependant acq modules to the latest versions
+* [MODFISTO-489](https://folio-org.atlassian.net/browse/MODFISTO-489) Clean up permissions
+* [MODFISTO-481](https://folio-org.atlassian.net/browse/MODFISTO-481) Separate credits from expenditures in mod-finance-storage
+* [MODFISTO-487](https://folio-org.atlassian.net/browse/MODFISTO-458) Prevent sql injection for budget_encumbrance_rollover script
```
Comment (azizbekxm) [inline NEWS.md:9]: -(https://folio-org.atlassian.net/browse/MODFISTO-487)

Comment (SerhiiNosko) [reply NEWS.md]: thanks, fixed

**Review by azizbekxm — APPROVED** [review]

**Pull request merged**